src/adagrams.js

```diff
@@ -1,8 +1,138 @@
 const Adagrams = {
   drawLetters() {
-    // Implement this method for wave 1
+    const letter_bank = {
+      A: 9,
+      B: 2,
+      C: 2,
+      D: 4,
+      E: 12,
+      F: 2,
+      G: 3,
+      H: 2,
+      I: 9,
+      J: 1,
+      K: 1,
+      L: 4,
+      M: 2,
+      N: 6,
+      O: 8,
+      P: 2,
+      Q: 1,
+      R: 6,
+      S: 4,
+      T: 6,
+      U: 4,
+      V: 2,
+      W: 2,
+      X: 1,
+      Y: 2,
+      Z: 1
+    };
+
+    let letters = [];
```

Comment:
weirdly enough, because `letters` never gets re-assigned (only modified), it can be a `const`